core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/AbstractCamelModelElement.java

```diff
@@ -54,6 +54,7 @@
 	public static final String PROPERTY_KEY_OLD_ID = "OLD_ID";
 	public static final String PROPERTY_KEY_NEW_ID = "NEW_ID";
 	public static final String PROPERTY_KEY_CAMEL_FILE = "CAMEL_FILE";
+	public static final String PROPERTY_LANGUAGENAME = "languageName";
```

Comment:
it is a property as displayed in Eclipse but in our model it is a parameter and in Camel Model it is an URI parameter.
Seems more coherent to call it PARAMETER_LANGUAGENAME in AbstractCamelModelElement, what do you think?